src/widgets/dashboardOverview.js

```diff
@@ -28,13 +28,40 @@ export default async function dashboardOverview(scopes) {
   const inPerson = duration.filter((report) => report.deliveryMethod.toLowerCase() === 'in-person').length
     .toString();
 
+  // Get Region.
+  const region = query['region.in'];
+
+  // Get Start and End Date.
+  const dateRange = query['startDate.win'];
+  if (!dateRange) {
+    return null;
+  }
+  const dates = dateRange.split('-');
```

Comment:
What here is requiring that `(query['startDate.win']` is a string with `-`? I'm asking because there's a cluster of errors in prod on the 29th with an error on a split in the `totalHrsAndGranteeGraph` widget.
```
TypeError: Cannot read property 'split' of undefined
    at _callee$ (/home/vcap/app/build/server/widgets/totalHrsAndGranteeGraph.js:60:31)
 ```
[new relic link](https://one.newrelic.com/launcher/nr1-core.explorer?platform[accountId]=3062577&platform[filters]=Iihkb21haW4gPSAnQVBNJyBBTkQgdHlwZSA9ICdBUFBMSUNBVElPTicpIg==&platform[timeRange][begin_time]=1627529700000&platform[timeRange][end_time]=1627696560000&platform[$isFallbackTimeRange]=false&pane=eyJuZXJkbGV0SWQiOiJlcnJvcnMtdWkub3ZlcnZpZXciLCJlbnRpdHlHdWlkIjoiTXpBMk1qVTNOM3hCVUUxOFFWQlFURWxEUVZSSlQwNThPRGMzTlRjd05Ea3gifQ==&sidebars[0]=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5hY3Rpb25zIiwic2VsZWN0ZWROZXJkbGV0Ijp7Im5lcmRsZXRJZCI6ImVycm9ycy11aS5vdmVydmlldyJ9LCJlbnRpdHlHdWlkIjoiTXpBMk1qVTNOM3hCVUUxOFFWQlFURWxEUVZSSlQwNThPRGMzTlRjd05Ea3gifQ==)

Reply:
This is pulled in from the the request to the BE. You can see where it gets added in the FE in Widget.js.

I don't think this should be an issue going forward as we prevent the closing of the dashboard date range selector until a valid date is entered. I have also worked with Josh to ensure we don't send any unnecessary calls before the page loads.

Let me know your thoughts.

Reply:
I think the API needs to validate the request params prior to acting on them. I'm seeing this as a problem all over the code base, we assume the FE has constructed a proper request and this leads to errors. It's too much to ask of future FE engineers that they know they can't change the date selector before a date is chosen because it'll trigger errors in the API response. But this is outside of the scope of this ticket and I don't want to hold it up for this. I'll write a new backlog ticket.

Reply:
https://ocio-jira.acf.hhs.gov/browse/TTAHUB-267